hiplan/base/task.py

```diff
@@ -1,8 +1,10 @@
 from hiplan.base.member import Member
 from typing import List
+from datetime import datetime
+import json
 
 class Task:
-    def __init__(self, title = "No title", description = '', deadline = None, members = None, goals = None, labels = None):
+    def __init__(self, title = "No title", description = '', deadline = datetime.now(), members = None, goals = None, labels = None):
```

Comment:
Nu pune `datetime.now()` in parametrii default. Este aceeasi poveste ca la  `memebers = []`.
Cand se parseaza fisierul de catre python (cand analizeaza codul ca sa stie ce ruleaza) o sa inlocuiasca `datetime.now()` cu valoarea returnata.  Adica daca tu rulezi la 12:55:12, indiferent cand apelezi `Task(deadline=None)`, `deadline` o sa fie 12:55:12